tests/test_opds_import.py

```diff
@@ -533,6 +535,24 @@ def test_extract_metadata(self):
         [failure] = failures.values()
         eq_(u"202: I'm working to locate a source for this identifier.", failure.exception)
 
+    def test_use_dcterm_identifier_as_id(self):
+        data_source_name = "Data source name " + self._str
+        importer = OPDSImporter(
+            self._db, collection=None, data_source_name=data_source_name
+        )
+        metadata, failures = importer.extract_feed_data(
+            self.feed_with_id_and_dcterms_identifier,
+            custom_identifier=ExternalIntegration.DCTERMS_IDENTIFIER
+        )
```

Comment:
This is missing the opposite test: importing this same feed with custom_identifier set to None. In that case `<id>` should be the primary identifier for both books.